### DISCUSSION TITLE
Can vite-plugin-ssr be used for React Native development

### DISCUSSION THREAD

**Discussion by haoynmail** [vikejs/vike#1079] [Discussion]
The client is a mobile device that uses React Native. Can it be developed using vite-plugin-ssr

**Comment by brillout** [comment]
I'm not that familiar with RN so I'm not sure what exactly is needed for making it work.

CC @alidcast how did your use of vite-plugin-ssr with React Native Web worked out? (https://github.com/brillout/vite-plugin-ssr/issues/513)

CC @natew which is working on a Vite React Native integration.

**Reply by haoynmail** [reply]
thank you
